src/cephadm/cephadm

```diff
@@ -6397,6 +6405,22 @@ def _parse_mem_usage(code: int, out: str) -> Tuple[int, Dict[str, int]]:
     return seen_memusage_cid_len, seen_memusage
 
 
+def _parse_cpu_perc(code: int, out: str) -> Tuple[int, Dict[str, str]]:
+    seen_cpuperc = {}
+    seen_cpuperc_cid_len = 0
+    if not code:
+        for line in out.splitlines():
+            (cid, cpuperc) = line.split(',')
+            try:
+                seen_cpuperc[cid] = cpuperc
+                if not seen_cpuperc_cid_len:
+                    seen_cpuperc_cid_len = len(cid)
+            except ValueError:
+                logger.info('unable to parse cpu percentage line\n>{}'.format(line))
+                pass
+    return seen_cpuperc_cid_len, seen_cpuperc
+
+
```

Comment:
It seems that the `code` variable in this method is useless. Wouldn't make more sense to check the code before calling this function, something like:

```
if not code:
     _parse_cpu_perc(out)

```

Reply:
> It seems that the `code` variable in this method is useless. Wouldn't make more sense to check the code before calling this function?, something like:
> 
> ```
> if not code:
>      _parse_cpu_perc(out)
> ```

Okay, but this is quincy backport, I'd suggest to apply the changes first in master and then backport it again! Wdyt @rkachach ?